assets/airports/airportLoadList.js

```diff
@@ -63,9 +63,9 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
         {
             icao: 'eddf',
             level: 'medium',
-            name: 'Frankfurt Airport',
+            name: 'Flughafen Frankfurt am Main',
```

Comment:
Did they really need the `am Main` at the end? Highly doubt people confused it with Frankfurt am Oder

Reply:
It is the official name though

Reply:
Yup, just found it funny